src/pkg/packaging/rpm/scripts/after_remove_host.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+echo "Removing dotnet host symbolic link"
+sudo unlink /usr/bin/dotnet
```

Comment:
also note that in many cases /bin/sh is bash. When /bin/sh is used, there may be surprises on distros where /bin/sh is more exotic - dash or original korn shell for example.